spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/IntegrationNode.java

```diff
@@ -59,6 +68,10 @@ public Stats getStats() {
 		return this.stats.isAvailable() ? this.stats : null;
 	}
 
+	public Map<String, Object> getProperties() {
+		return this.properties;
```

Comment:
M-m-m. Can't we simplify the life for Jackson and for client afterward and return `null` here if `properties` is empty?
I'm sure that we will add eventually more and more here, but there may be some cases when it is still empty.